### PR TITLE
[TIMOB-24922] Update to windowslib 0.5.1

### PR DESCRIPTION
**Pull request by ewanharris**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24922

Includes the following
- appcelerator/windowslib/pull/71 - [TIMOB-24922](https://jira.appcelerator.org/browse/TIMOB-24922) Fix cert generation on VS2017 
- appcelerator/windowslib/pull/70 - [TIMOB-24668](https://jira.appcelerator.org/browse/TIMOB-24668) Update VS2017 detection logic to prevent error with VS2015
- Try to make wptool.connect poll emulator status - no ticket?
- appcelerator/windowslib/pull/67 - [TIMOB-24422](https://jira.appcelerator.org/browse/TIMOB-24422) Windows: Fails to install dependencies for app on Windows 10 Phone Emulator

This is needed to be merged so I can continue with VS 2017 testing

### CODE REVIEW

**Review by garymathews — APPROVED** [review]
LGTM

FT: **PASS**

**Comment by ewanharris** [comment]
This will probably be replaced by an update to 0.5.2 once appcelerator/windowslib/pull/72 is merged so hold off on merging and I'll close if so

**Comment by ewanharris** [comment]
Closing, will push a new PR with 0.5.2

**Pull request closed without merging**
